### ISSUE TITLE
The World Transformed App

### ISSUE THREAD

**Issue by ruthcsberry** [PeoplesMomentum/projects#28]
Includes:
- Email signup box
- Schedule of events
- Map of venues
- Ticketing system

![splash screen](https://user-images.githubusercontent.com/28708398/28064235-c1ea7662-662b-11e7-9b42-12043fce8edd.jpg)

![programme screen 1](https://user-images.githubusercontent.com/28708398/28064232-c1e679ae-662b-11e7-9bea-bdd7c2b476f0.jpg)

![programme screen 2](https://user-images.githubusercontent.com/28708398/28064233-c1e9323e-662b-11e7-8af1-2494db1138c4.jpg)

![map screen](https://user-images.githubusercontent.com/28708398/28064231-c1e47f6e-662b-11e7-82a3-d0435a1cd862.jpg)

![tickets screen 1](https://user-images.githubusercontent.com/28708398/28064234-c1ea1550-662b-11e7-9419-e647f126b40d.jpg)

![tickets screen 2](https://user-images.githubusercontent.com/28708398/28064236-c1eb30fc-662b-11e7-963e-6b6d12944bfb.jpg)

**Comment by LesleyJC** [comment]
Hi,

I have a free day tomorrow if I can help with anything graphic design wise? 

Thanks

Lesley

**Comment by paulholmes82** [comment]
Do you need help with nailing down the requirements?

**Comment by LesleyJC** [comment]
Hi Paul,

Yes. 

It's my first time on the system following the Digital Hub I registered for. I'm not sure what various things are such as Github, Kanban and Slack. It'll all probably be much clearer after I've been to the event on Saturday :)

For the app job I messaged about I'd need fonts, any graphics you use, size, format, which software you'd like it producing in etc. 

I'm a freelance designer and have worked on mobile apps before and I've produced the designs in Indesign. 

Thanks

Lesley
